<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBCodeInspectionDialog</class>
 <widget class="QDialog" name="AddBCodeInspectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxQuilityGrade">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="maximum">
    <double>9999.989999999999782</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>閉じる</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSet">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>設定</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>品質しきい値</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxCheckType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>読み取り 2D</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>品質チェック</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>読み取り 1D</string>
    </property>
   </item>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>341</width>
     <height>211</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Read">
    <widget class="QTableWidget" name="tableWidgetBarcodeType">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>181</width>
       <height>181</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Code</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>181</width>
       <height>30</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>読取コード</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidgetBarcodeOrientation">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>141</width>
       <height>131</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Code</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>0</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>バーコード方向</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxBarcodeIsOnlyDigit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>160</y>
       <width>141</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>数値のみ</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Quality">
    <widget class="QPushButton" name="pushButtonSub">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>150</y>
       <width>41</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSave">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>110</y>
       <width>91</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>231</width>
       <height>29</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>品質割当表</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAdd">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>150</y>
       <width>41</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidgetGradeList">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>231</width>
       <height>171</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Grade</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>最低品質値</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pushButtonLoad">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>70</y>
       <width>91</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>91</width>
       <height>29</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>参考データ</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
